api/src/main/java/org/openmrs/Location.java

```diff
@@ -303,7 +303,8 @@ public Location hydrate(String locationId) {
 			return Context.getLocationService().getLocation(Integer.valueOf(locationId));
 		}
 		catch (Exception e) {
-			return new Location();
+			
```

Comment:
Remove unnecessary blank line here and don't you think a more descriptive and user friendly error message should be appended rather than just throw the stack trace?